WORKSPACE

```diff
@@ -168,7 +171,7 @@ bind(
 #
 git_repository(
     name = "io_bazel_rules_pex",
-    commit = "6af30588d4f11cafcb744c50935cc37f029e6e7f",
+    commit = "0c5773db01ab8aeb3ae749b2fc570749b93af41f",
```

Comment:
please add comment on to state the date of this SHA

Reply:
Done.